opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/jms/JmsArtemisManualServerTest.java

```diff
@@ -13,17 +13,20 @@
  */
 package io.opentracing.contrib.spring.cloud.jms;
 
-import static org.awaitility.Awaitility.await;
-import static org.junit.Assert.assertEquals;
+import static org.awaitility.Awaitility.*;
+import static org.junit.Assert.*;
```

Comment:
.* ?

Reply:
yeah, wildcard imports aren't good

Reply:
Somebody should then fix code-style plugin. ;-)
As this all passed. :-)